modules/integration_azure_firewall/conf/03-health-status.yaml

```diff
@@ -0,0 +1,27 @@
+module: firewall
+name: "Health state"
+
+transformation: false
+aggregation: false
+filtering: "filter('resource_type', 'Microsoft.Network/azureFirewalls') and filter('primary_aggregation_type', 'true')"
+tip: "https://docs.microsoft.com/en-us/azure/firewall/logs-and-metrics#metrics"
```

Comment:
I would use a full message to explain why this doc is useful (if you provide a link which is enough itself so may be `runbook_url` is better suited)